pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApi2Test.java

```diff
@@ -924,6 +925,23 @@ public void testUpdatePartitionedTopicProperties() throws Exception {
         Assert.assertEquals(properties.size(), 2);
         Assert.assertEquals(properties.get("key1"), "value11");
         Assert.assertEquals(properties.get("key2"), "value2");
+
+        // create topic without properties
+        admin.topics().createPartitionedTopic(topicNameTwo, 2);
+        properties = admin.topics().getProperties(topicNameTwo);
+        Assert.assertNull(properties);
+        // remove key of properties on this topic
+        admin.topics().removeProperties(topicNameTwo, "key1");
+        Assert.assertNull(properties);
+        properties = new HashMap<>();
+        properties.put("key1", "value1");
+        properties.put("key2", "value2");
+        admin.topics().updateProperties(topicNameTwo, properties);
+        properties = admin.topics().getProperties(topicNameTwo);
+        Assert.assertEquals(properties.size(), 2);
+        admin.topics().removeProperties(topicNameTwo, "key1");
+        properties = admin.topics().getProperties(topicNameTwo);
+        Assert.assertEquals(properties.size(), 1);
```

Comment:
Make sure we removed the correct `key`
```suggestion
        Assert.assertEquals(properties.size(), 1);
        Assert.assertNotNull(properties.get("key2"));
        Assert.equals(properties.get("key2"), "value2");
        
```